packages/indy-vdr/src/dids/didIndyUtil.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  AriesFrameworkError,
+  convertPublicKeyToX25519,
+  DidDocument,
+  DidDocumentBuilder,
+  JsonTransformer,
+  TypedArrayEncoder,
+} from '@aries-framework/core'
+import { mergeWith, compact, transform, isUndefined, isEqual, isObject, isArray } from 'lodash'
```

Comment:
I think dependencies for the framework are already heave, so I'm not too much of a fan of adding another. Espeically lodash is quite big, and hasn't had releases for the past 2 years, which I'm not sure we should start depending on now.

There's been a lot of feature additions to JS over the years that have add native utils for what lodash provides. There's even a whole website dedicated to it: https://you-dont-need.github.io/You-Dont-Need-Lodash-Underscore/#/

If you really think we need to use lodash in this case, I think it would be good to at least not import from the main bundle as that will pull in the whole lodash library (which is really big).

Reply:
Yes, I don't like to add dependencies for such specific tasks. In this case I didn't care so much because I've seen that lodash is already retrieved as an indirect dependency of @aries-framework/core.  Sorry for my ignorance here, but is there any difference in practice in this case?

Anyway I'll try to get some stand-alone implementations for merge/diff.